unsafe.go

```diff
@@ -16,10 +16,11 @@ func UnsafeBytesToString(b []byte) string {
 
 // UnsafeStringToBytes converts string to source without copying
 func UnsafeStringToBytes(s string) []byte {
```

Comment:
Wait, why do we need this code, exactly? @Nixolay you've said that it is still used somewhere, but I can't find any usages except in tests that test this code.

Reply:
OK, my bad, ignore this comment.